README.md

```diff
@@ -62,7 +66,7 @@ To execute the Terraform script:
 
        set_tenant_level_permissions = true
 
-       parent_management_group_name = "ID of the parent management group in a tenant"
+       parent_management_group_id = "ID of the parent management group in a tenant"
```

Comment:
in case of tenant integration, it usually is the root management group right? If that is so, then the name `parent management group` is a little ambiguous. Can we call it the `root management group`?